client/state/index.js

```diff
@@ -31,6 +32,7 @@ import reader from './reader/reducer';
 export const reducer = combineReducers( {
 	plugins,
 	application,
+	document,
```

Comment:
Ok, this does a great job of highlighting the danger in naming this subtree `document`, as I'm not confident what's truly being passed to `combineReducers` here: `window.document` or the `document` reducer?

/cc @mtias : Any thoughts on naming? `state.page` ? `state.screen` ? Top-level or under `ui` ?